lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/NativeInterop.java

```diff
@@ -35,11 +35,7 @@
 
   public static native void clrBufferedLog(final int level, final String message);
 
-  public static native void callClrSystemOnStartHandler(
-      final String dateTime,
-      final String httpServerPortNumber,
-      final BridgeHandlerManager bridgeHandlerManager,
-      final EvaluatorRequestorBridge javaEvaluatorRequestorBridge);
+  public static native void callClrSystemOnStartHandler();
```

Comment:
This is related to your other comment. If we use the Java-side time as the driver start time, then we need the time argument here. If we don't, then no information regarding time needs to be passed from Java to .NET and thus we can remove the time argument.